fastlane/lib/fastlane/runner.rb

```diff
@@ -258,13 +258,17 @@ def execute_action(method_sym, class_ref, arguments, custom_dir: nil, from_actio
         raise e
       rescue FastlaneCore::Interface::FastlaneError => e # user_error!
         FastlaneCore::CrashReporter.report_crash(exception: e, action: method_sym)
-        collector.did_raise_error(method_sym)
+        unless e.fastlane_crash_came_from_custom_action? || e.fastlane_crash_came_from_plugin?
```

Comment:
I'm a little confused when using `unless` in combination with `||`, can we update that to something a little easier to read? :)